cloudformation/apps/author.yaml

```diff
@@ -93,6 +101,31 @@ Conditions:
 
 Resources:
 
+  AemAuthorSyncDelayAlarmTopic: 
+    Type: AWS::SNS::Topic
+    Properties: 
+      Subscription: 
+        - Endpoint: !Sub ${AemAuthorSyncDelayAlarmContactEmailParameter}
+          Protocol: email
+      TopicName: !Sub ${AemAuthorSyncDelayAlarmTopicNameParameter}
+
+  AemAuthorSyncDelayAlarm:
```

Comment:
for naming consistency with other resources. remove the Aem from 'AemAuthorSyncDelayAlarm'  the Author part assumes that it is aem.